plugins/modules/version_update_info.py

```diff
@@ -22,12 +22,30 @@
 seealso:
   - module: scale_computing.hypercore.version_update
   - module: scale_computing.hypercore.version_update_status_info
+options:
+  select:
+    description:
+      - List desired version update from list of updates
+      - If C(next), then version with the lowest number in the list of available updates is returned
+      - If C(latest), then version with the highest number in the list of available updates is returned
+    type: str
+    choices: [ next, latest ]
```

Comment:
I guess sometimes it is still nice to be able to list all available updated - as `all` choice. Main use case would be debugging, I guess.

Why not still returning the complete list, but sort it before it is returned? Then later in playbook you can `{{ (updates_result.records | last).uuid }}` or `{{ (updates_result.records | first).uuid }}`. I would sort them in ascending order (0,1,2,...) - is sort of natural.

Reply:
Comment was invalid, `records` was unmodified, only `record` output was added.